model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -132,10 +132,18 @@ object AddressResponseAddress {
     * @param other
     * @return
     */
-  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress =
+  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress = {
+
+    val poBoxNumber = if (other.poBoxNumber.isEmpty) "" else s"PO BOX ${other.poBoxNumber}"
+
```

Comment:
Why not interpolate like so:
`s"$other.departmentName $other.organizationName $other.subBuildingName".split(" ").mkString(" ")`